libs/minilibx/hooks.md

```diff
@@ -69,6 +69,44 @@ a key. As you can see, we register a hook function with `mlx_key_hook`. However
 in the background it simply calls the function `mlx_hook` with the appropriate
 X11 event types. We will discuss this in the next chapter.
 
+## Hooking into mouse events
+
+Also, you can hook mouse events.
+
+```c
+mlx_mouse_hook(vars.win, mouse_hook, &vars);
+```
+
+<details>
+  <summary>Mouse buttons</summary>
+
+<table>
+<tr>
+<th align="center">Schema</th>
+<th align="center">Buttons</th>
+</tr>
+<tr>
+<td>
+
+![Mouse buttons](https://user-images.githubusercontent.com/92152391/147275282-90ff2fa2-810d-4c7d-b7bd-0c72d6192d7f.png)
+
+</td>
+<td>
+
+  - Left click: 1
+  - Right click: 2
+  - Middle click: 3
+  - Scroll up: 4
+  - Scroll down : 5  
+
+</td>
+</tr>
+</table>
+  
+</details>
+
+
+
```

Comment:
Can you create a markdown table for this?

Reply:
We cannot make a list in markdown table

Reply:
@harm-smits I used HTML img with right align instead html table

![image](https://user-images.githubusercontent.com/92152391/147408382-6a6a1868-889c-4f4a-a31a-08933ad0df80.png)